usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -620,3 +621,35 @@ def subaward_annotations():
         "prime_award_latest_action_date_fiscal_year": FiscalYear("award__latest_transaction__action_date"),
     }
     return annotation_fields
+
+
+def disaster_recipient_annotations():
+    annotation_fields = {
+        "award_obligations": Coalesce(
+            Sum(
+                "award__financial_set__transaction_obligated_amount",
+                filter=Q(
```

Comment:
I think `filter_limit_to_closed_periods()` is needed here

Reply:
Looking at the `award_delta_view` it does not appear that we care about `latest_closed_period_per_fy` when dealing with obligation. Happy to add it and see what the results look like though.

Reply:
This?
https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/database_scripts/etl/award_delta_view.sql#L229-L233

Reply:
Yeh I saw the left join and missed the inner join. Need to fix loans and then I will test both in Sandbox.

Reply:
@tony-sappe Just tested in Sandbox. Adding the filter to the `award_obligations` does not result in the same data.

Reply:
@sethstoudenmier that appears to be the wrong filter function. Don't need to limit to only the latest per FY, just all closed